tensorboard/webapp/webapp_data_source/tb_feature_flag_data_source_types.ts

```diff
@@ -43,7 +43,7 @@ export const ENABLE_COLOR_GROUP_BY_REGEX_QUERY_PARAM_KEY =
 
 export const ENABLE_DARK_MODE_QUERY_PARAM_KEY = 'darkMode';
 
-export const ENABLE_LINK_TIME_PARAM_KEY = 'enableLinkTime';
+export const ENABLE_LINKED_TIME_PARAM_KEY = 'enableLinkedTime';
```

Comment:
I agree, this is a bit better too. It's going to give you a little trouble with the copybara import but perhaps you could just do a sync immediately after merging the PR?

Also, don't forget to tell James about it!